doc: Add what is TTS section (#190)
TheThingsIndustries/lorawan-stack-docs: doc/content/getting-started/what-is-tts/_index.md

---
title: "What Is The Things Stack?"
description: ""
weight: -2
---

Welcome to {{% tts %}}! In this section, we help you to understand how to use {{% tts %}}, who makes it, and common acronyms like TTS, TTI, and TTN.

<!--more-->

## What is {{% tts %}}?

{{% tts %}} is an enterprise grade LoRaWAN network server, built on an [open source](https://github.com/TheThingsNetwork/lorawan-stack) core. {{% tts %}} allows you to build and manage LoRaWAN networks on your own hardware or in the cloud.

To get started running {{% tts %}} on your own, see the [installation section]({{< ref "/getting-started/installation" >}}). To inquire about a worry-free hosted solution, [get in touch with us](mailto:info@thethingsindustries.com).

## Who makes {{% tts %}}?

{{% tts %}} is developed and maintained by [The Things Industries](https://thethingsindustries.com/).

The Things Industries offers several types of deployments of {{% tts %}}, for example as a managed cloud service, or with additional proprietary features.

## What are TTI, TTN, and TTS?

You might see abbreviations like TTN, TTS, and TTI tossed around in the forums and in other places. Here's what they mean:

> ## TTS: {{% tts %}}

Reviewer: ```suggestion
### TTS: {{% tts %}}
```

> TTS is of course {{% tts %}}, the network server itself. {{% tts %}} is technically version 3 of our network server implementation.

Reviewer: ```suggestion
TTS is {{% tts %}}, the LoRaWAN Network Server stack. {{% tts %}} is currently at version 3 of a Network Server implementation, and is therefore also informally known as V3.
```

> ## TTN: The Things Network

Reviewer: ```suggestion
### TTN: The Things Network
```

> TTN is [The Things Network](thethingsnetwork.org), which is a public community LoRaWAN network available for anyone to use and extend with their own gateways. The Things Industries manages The Things Network. The Things Network has historically run version 2 of the network server implementation, but this is changing soon.

Reviewer: ```suggestion
TTN is [The Things Network](thethingsnetwork.org), which is a public community LoRaWAN network available for anyone to use and extend with their own gateways. The Things Network is operated by community contributors, including The Things Industries. The Things Network has historically run version 2 of the Network Server implementation, but is planned to upgrade to version 3.
```

> In 2021, The Things Network will transition to {{% tts %}} v3. For a guide on migrating from {{% ttnv2 %}} to {{% tts %}}, see [here]({{ ref "/getting-started/migrating/major-changes"}}).

Reviewer: ```suggestion
In 2021, The Things Network will transition to {{% tts %}} V3. For a guide on migrating from {{% ttnv2 %}} to {{% tts %}}, see [here]({{ ref "/getting-started/migrating/major-changes"}}).
```

> ## TTI: The Things Industries

Reviewer: ```suggestion
### TTI: The Things Industries
```

> TTI is The Things Industries. We're the company primarily responsible for development of {{% tts %}}, writing documentation and hosting The Things Network. The Things Industries also offers cloud hosted and on premise private LoRaWAN networks with additional enterprise features, and premium support plans for enterprise clients. If you're interested in building your own guaranteed uptime enterprise grade LoRaWAN network, send us an [email](mailto:info@thethingsindustries.com).

Reviewer: ```suggestion
TTI is The Things Industries: the company primarily responsible for development of {{% tts %}} and writing documentation. The Things Industries also offers cloud hosted and on-premises private LoRaWAN networks with additional enterprise features, and premium support plans for enterprise clients. If you're interested in building your own guaranteed uptime enterprise grade LoRaWAN network, send us an [email](mailto:info@thethingsindustries.com).
```
